content/guides/models/registry/configure_registry.md

```diff
@@ -7,107 +7,121 @@ title: Configure registry access
-3. Click on the gear icon on the upper right hand corner.
-4. Scroll to the **Registry members and roles** section and type in the username of the member you want to remove.
-5. Click the **Delete** button.
 
+### Inherited permissions
 
+A user's permission in a registry depends on the highest level of privilege assigned to that user, whether through a team or individual. 
 
-## Registry visibility types
+For example, suppose a registry admin adds a user called Nico to Registry A and assigns them a `Viewer` registry role. The registry admin then adds a team called Foundation Model Team to Registry A and assigns Foundation Model Team a `Member` registry role. Nico is a member of Foundation Model Team.
 
-There are two registry visibility types: restricted or organization visibility. The following table describes who has access to the registry by default:
+Since Nico is a member of Foundation Model Team, W&B grants Nico `Member` registry role permissions because `Member` registry roles have higher level of permissions than `Viewer`. 
 
-| Visibility | Description | Default role | Example |
-| --- | --- | --- | --- |
-| Organization | Everyone in the org can access the registry. | By default, organization administrators are an admin for the registry. All other users are a viewer in the registry by default. | Core registry |
-| Restricted   | Only invited org members can access the registry.| The user who created the restricted registry is the only user in the registry by default, and is the organization's owner. | Custom registry or core registry |
+The proceeding table shows the highest level of permission in the event of a conflict:
 
+| Registry role A | Registry role B  | Inherited registry role |
+| ------ | ------ | ------ | 
+| Viewer | Viewer | Viewer |
+| Member | Viewer | Member |
+| Admin  | Viewer | Admin  | 
 
-## Restrict visibility to a registry
-<!-- Who can do this? -->
-Restrict who can view and access a custom registry. You can restrict visibility to a registry when you create a custom registry or after you create a custom registry. A custom registry can have either restricted or organization visibility. For more information on registry visibilities, see [Registry visibility types]({{< relref "./configure_registry.md#registry-visibility-types" >}}).
+W&B displays the highest level of permissions next to the name of the user in the event of a conflict.
 
-<!-- | Visibility | Description |
-| --- | --- |
-| Organization | Anyone in the organization can view the registry. |
-| Restricted   | Only invited organization members can view and edit the registry.|  -->
+{{< img src="/images/registry/role_conflict.png" alt="A user inherits a Member role because they are part of a team." >}}
 
-The following steps describe how to restrict the visibility of a custom registry that already exists:
+For example, in the preceding image Ishita inherits `Member` role privileges because she is a member of smle-reg-team-1.
```

Comment:
I think Member should be bold rather than code font, and should probably not refer to an actual employee and an actual role in the W&B org, right?